apps/src/lab2/views/components/panelContainer.module.scss

```diff
@@ -31,10 +31,21 @@ $default-spacing: 2px;
   border-bottom: 1px $light_gray_200 solid;
 }
 
+.panelContainerHeaderItem-light {
```

Comment:
nit: I think the preference is to use the design system color (ones under [here ](https://github.com/code-dot-org/code-dot-org/blob/staging/shared/css/color.scss#L190-L191)) `$light_black` and `$light_white` in favor of `$neutral_dark` and `$neutral_white`

Reply:
(they are the same color codes)

Reply:
updated!